tableauserverclient/server/endpoint/views_endpoint.py

```diff
@@ -105,7 +105,7 @@ def csv_fetcher():
     def _get_view_csv(self, view_item, req_options):
         url = "{0}/{1}/data".format(self.baseurl, view_item.id)
 
-        with closing(self.get_request(url, parameters={"stream": True})) as server_response:
```

Comment:
I don't think this is right, the 'stream=true' is an option for the `requests` library, not something that gets included in the url string, which is what `apply_query_parameters` does.